mock/py/mockbuild/plugins/root_cache.py

```diff
@@ -58,7 +58,11 @@ def __init__(self, plugins, conf, buildroot):
         plugins.add_hook("postchroot", self._rootCachePostShellHook)
         plugins.add_hook("postyum", self._rootCachePostShellHook)
         self.exclude_dirs = self.root_cache_opts['exclude_dirs']
-        self.exclude_tar_cmds = ["--exclude=" + item for item in self.exclude_dirs]
+        if self.config['tar'] == 'bsdtar':
+            anchor = '^'
+        else:
+            anchor = ''
+        self.exclude_tar_cmds = ["--exclude=" + anchor + item for item in self.exclude_dirs]
 
```

Comment:
Don't you want to make some helper method, say `tar_exclude_option(item)` returning the proper exclude option at one place?